docs/developer-guide/dev-cluster-with-kind.md

```
@@ -118,8 +123,18 @@ This will:
-3. Transfer the ocm-demo component via OCM
-4. Clean up the port-forward
+3. Transfer the ocm-demo component via OCM (using `test/fixtures/e2e/ocmconfig`,
+   which trusts the cluster CA)
+4. Clean up the port-forward (always, via a trap)
+
+Since `make dev-cluster` deploys solar-discovery in scan mode with the discovery
+Zot already registered, you do not apply a `Registry` yourself. A few seconds
+after the transfer the discovery worker scans the registry and creates the
+`Component` and `ComponentVersion` in the `solar-system` namespace:
+
+```bash
```

markdownlint-cli2 (0.23.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)